doc/avocado_predictors_report.Rmd

```diff
@@ -82,49 +83,72 @@ However, region and month are categorical variables that have numerous levels. T
 ![](../doc/img/correlation_matrix.png)
 **Figure 5.** Correlation matrix of continuous features.
 
-Overall, there is fairly high collinarity between many of the predictors. This was expected, since they all deal with volume of avocados sold, be it by PLU code, bag type or total volume. In particular, `total_bags` and `total_volume` were expected to be highly correlated to other predictors that were sub-quantities of these totals. Due to the high correlation, including all these predictors in a prediction model would probably lead to overfitting.
+Overall, there is fairly high collinarity between many of the predictors. This was expected, since they all deal with volume or number of avocados sold, be it by PLU code, bag type or total volume. In particular, `total_bags` and `total_volume` were expected to be highly correlated to other predictors that were sub-quantities of these totals. Due to the high correlation, including all these predictors in a prediction model would probably lead to overfitting.
+
+To verify the result from the correlation matrix above, we also computed the variance inflation (VIF) scores from the `car` package.
 
-To verify the result from the correlation matrix above, we also computed the variance inflation (VIF) scores from the car package.
 ![](../doc/img/collinearity.png)
 **Figure 6.** Variance inflation scores of continuous features.
 
-This suggests extremely high collinearity for these variables in a linear model. Therefore, we will be careful about using these features as they are probably not very good predictors of the average avocado price.
+The high VIF scores suggest extremely high collinearity for these variables in a linear model. Therefore, we will be careful about using these features as they are probably ineffective predictors of the average avocado price.
 
 ## Random Forest Feature Importances
 
-Lastly, we fitted a random forest regressor model using the features that we determined as significant from the analysis above (`region`, `type`, and `month`). We used one hot encoding on these categorical features and used cross validation to determine the optimal hyperparameters. We found that our top predictor of avocado prices is `type` (i.e. whether the avocado is organic or convenional).
+Lastly, we fitted a random forest regressor model using the features that we determined as significant from the analysis above (`region`, `type`, and `month`). We used one hot encoding on these categorical features and used randomized cross validation to determine the optimal hyperparameters, maximum depth and number of estimators. We calculated the cross-validation scores to determine how well our model was performing.
```

Comment:
Hmm, can we say "cross-validation" scores? How about "We calculated the average (validation) scores using cross validation..."

Reply:
I like this! I will change it.

Reply:
I see you used this terminology in other places too. If that's how it's used, I'm ok with leaving as is. I was just curious as I've not come across it worded like that before.